src/test/java/reposense/template/GitTestTemplate.java

```diff
@@ -34,13 +32,13 @@ public void before() {
     }
 
     @BeforeClass
-    public static void beforeClass() throws GitClonerException {
+    public static void beforeClass() throws Exception {
```

Comment:
Instead of a generic Exception, perhaps throw the individual exceptions instead? So that we can trace back if error occurs.